### PR TITLE
Improve documentation for App Insights/Log Analytics purge

### PR DESCRIPTION
**Pull request by alexeldeib**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json [hunk 1]
```diff
@@ -539,7 +539,7 @@
           "type": "string"
         },
         "operator": {
-          "description": "A query operator to evaluate over the provided column and value(s).",
+          "description": "A query operator to evaluate over the provided column and value(s). Supported operators are ==, =~, in, in~, >, >=, <, <=, between, and have the same behaviour as they would in a KQL query.",
```
Comment (nschonni) [inline specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json]:
```suggestion
          "description": "A query operator to evaluate over the provided column and value(s). Supported operators are ==, =~, in, in~, >, >=, <, <=, between, and have the same behavior as they would in a KQL query.",
```

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2015-03-20/OperationalInsights.json [hunk 2]
```diff
@@ -1412,7 +1412,7 @@
           "type": "string"
         },
         "operator": {
-          "description": "A query operator to evaluate over the provided column and value(s).",
+          "description": "A query operator to evaluate over the provided column and value(s). Supported operators are ==, =~, in, in~, >, >=, <, <=, between, and have the same behaviour as they would in a KQL query.",
```
Comment (nschonni) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2015-03-20/OperationalInsights.json]:
```suggestion
          "description": "A query operator to evaluate over the provided column and value(s). Supported operators are ==, =~, in, in~, >, >=, <, <=, between, and have the same behavior as they would in a KQL query.",
```

**Review by nschonni — COMMENTED** [review]
behaviour -> behavior
The original spelling is correct for me, but I think the docs are generally US spelling

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
While you are at it, can you also fix the model validation failures. I understand those failures are not related to your changes, but take this opportunity to fix the examples.
1) Add required parameter 'savedSearchId' in the example 'WorkspacesDeleteSavedSearches'
2) Add required parameter 'savedSearchId' in the example 'WorkspacesSavedSearchesCreateOrUpdate'
There are additional similar errors take a look at
https://dev.azure.com/azure-sdk/public/_build/results?buildId=25819

**Comment by alexeldeib** [comment]
Sure, these look simple enough to fix. cc @yoramsinger as FYI since I'm touching saved search (I think this is you Yoram?).

**Comment by yoramsinger** [comment]
Look fine.

**Comment by alexeldeib** [comment]
Yeah, ended up being very simple. @shahabhijeet let me know if there's anything else

**Review by shahabhijeet — APPROVED** [review]

**Comment by alexeldeib** [comment]
@shahabhijeet I noticed this hasn't merged, seems like a bug in the .NET test pipeline handling multiple RPs?

```
Successfully fetched all files modified in the REST spec PR. Calculating the resource provider name.
15:55:57      1>c:\Jenkins\workspace\DotNetSwaggerToSdkCI\scripts\stostools\SToS.proj(17,9): error : Exception: Cannot handle multiple RPs: applicationinsights/resource-manager, operationalinsights/resource-manager
```

**Comment by alexeldeib** [comment]
@nschonni @shahabhijeet any update here? Circling back because I checked the docs and realized the PR still hadn't merged.. Our team receives customer questions on the allowed operators in this doc fairly regularly, would love to get this change merged.

**Comment by nschonni** [comment]
Sorry, I'm not an MS employee. There is an internal link in the PR template that might have steps/contacts for getting something pushed forward, but I can't see whet it says 😄

**Comment by alexeldeib** [comment]
Sorry @nschonni! I assumed we simply hadn't crossed paths yet :smile:

@lmazuel @sarangan12 could one of you please take a look at the above error?

**Pull request merged**